.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,23 @@
+name: Shellcheck
+
+# shellcheck gives warnings and suggestions for bash/sh shell scripts.
+# https://github.com/koalaman/shellcheck
+
+permissions:
+  contents: read
+
+on: [workflow_dispatch, pull_request, push]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    name: Shellcheck - Lint
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out ockam repository
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+      - name: Run ShellCheck
+        uses: ludeeus/action-shellcheck@master
```

Comment:
Let's pin this workflow to latest master SHA `b2bbefc2e6b9dcbc6355b85e366c9e55bf8d57e1`
```
uses: ludeeus/action-shellcheck@b2bbefc2e6b9dcbc6355b85e366c9e55bf8d57e1
```

Reply:
Fixed.